### ISSUE TITLE
Rollout of IPv6 in the freifunkh net

### ISSUE THREAD

**Issue by lemoer** [freifunkh/ansible-configs#20]
I open this issue to plan the rollout of IPv6 in the freifunkh. Infos will be following.

**Comment by lemoer** [comment]
I think we should test freifunkh/site#7, so i'll add the blocked tag here, until this is done.

**Comment by lemoer** [comment]
Or at least we should have a look, **whether** we need this package before the rollout.

**Comment by lemoer** [comment]
I'll ask Hostway for PA IPv6 space, which we could use. I think we should begin at Hostway.

**Comment by jue89** [comment]
Routing works. rDNS kind of works (i.e. * resolves to client.ffh.zone). :)

What must be done:
 * rDNS zu All-Knowing-DNS + some manually entered rDNS
 * Routing a /64 net to some super node
 * Announce the net
 * Info mail to the mailing list to remind the users to review their firewall settings

**Comment by lemoer** [comment]
According to the firewall:
- We should add a possible to the simple_firewall rule in freifunkh/ansible to only allow forwarding for a specific net (`/64`). So we don't throw wrong addresses to the internet.

**Comment by jue89** [comment]
You are talking about async routing, lemoer? Async routing occurs easily in multi-homed setups.

Maybe just filter all non-TCP traffic? So we ensure that no reflection attacks can be carried out abusing Freifunk. Afaik reflection attacks are not possible with TCP due to the stateful design. Furthermore, our firewall has less packets to filter if TCP can just pass through. And we do not want to bug the user if his OS makes bullshit.

**Comment by lemoer** [comment]
Hm. Someone (hopefully Hostway) will filter the packets which are not from our `/48`. (E.g. someone tries to send from our ULA-addresses to the internet). To make this in a clean way, we should filter it by ourself, not leaving this job to our upstreams.

**Comment by jue89** [comment]
Good point with ULA addresses. Maybe we just should ask them what they want us to do.

**Comment by lemoer** [comment]
Maybe we should announce the `-O` flag with radv packets and add a stateless ipv6 dhcp server to announce the ipv6 dns server, since windows clients earlier than windows 10.x are not accepting the RRDNS records in the radv.

[Reference](https://forum.darmstadt.freifunk.net/t/administratives-tagebuch/32/53)

**Comment by 1977er** [comment]
#YOLO

**Comment by 1977er** [comment]
It's done, isn't it? @lemoer @jue89

**Comment by lemoer** [comment]
I think we should leave this open, as we currently only add routes manually via terminal.
